src/Microsoft.Extensions.Logging.Testing/Xunit/LoggedTestInvoker.cs

```diff
@@ -57,12 +57,15 @@ protected override object CreateTestClass()
             if (testClass is LoggedTest loggedTestClass)
             {
                 var classType = loggedTestClass.GetType();
-                var logLevelAttribute = TestMethod.GetCustomAttribute(typeof(LogLevelAttribute)) as LogLevelAttribute;
+                var logLevelAttribute = TestMethod.GetCustomAttribute<LogLevelAttribute>() as LogLevelAttribute;
                 var testName = TestMethodArguments.Aggregate(TestMethod.Name, (a, b) => $"{a}-{(b ?? "null")}");
 
                 // Try resolving ITestOutputHelper from constructor arguments
                 loggedTestClass.TestOutputHelper = ConstructorArguments?.SingleOrDefault(a => typeof(ITestOutputHelper).IsAssignableFrom(a.GetType())) as ITestOutputHelper;
 
+                var useShortClassName = TestMethod.DeclaringType.GetCustomAttribute<ShortClassNameAttribute>()
```

Comment:
Test?

Reply:
Hmm this is going to be interesting to test...